prov/efa/src/efa_av.c

```diff
@@ -524,18 +548,32 @@ int efa_av_insert(struct fid_av *av_fid, const void *addr,
 }
 
 static int efa_av_lookup(struct fid_av *av_fid, fi_addr_t fi_addr,
```

Comment:
This function is never used internally in the EFA provider, correct?

Reply:
Yes.

It is used when application call `efa_av_lookup`

Reply:
Yes.

This is used when application call `fi_av_lookup`. EFA provider itself does not use it.